docs/standard/datetime/system-text-json-support.md

```diff
@@ -150,90 +124,89 @@ then parsed using <xref:System.DateTimeOffset.ParseExact(System.String,System.St
 
 | Component       | Format                      | Description                                                                     |
 |-----------------|-----------------------------|---------------------------------------------------------------------------------|
 | Year            | "yyyy"                      | 0001-9999                                                                       |
 | Month           | "MM"                        | 01-12                                                                           |
-| Day             | "dd"                        | 01-28, 01-29, 01-30, 01-31 based on month/year                                  |
+| Day             | "dd"                        | 01-28, 01-29, 01-30, 01-31 based on month/year.                                 |
 | Hour            | "HH"                        | 00-23                                                                           |
 | Minute          | "mm"                        | 00-59                                                                           |
 | Second          | "ss"                        | 00-59                                                                           |
-| Second fraction | "FFFFFFF"                   | Minimum of one digit, maximum of 16 digits                                      |
-| Time offset     | "K"                         | Either "Z" or "('+'/'-')HH':'mm"                                                |
-| Partial time    | "HH':'mm':'ss[FFFFFFF]"     | Time without UTC offset information                                             |
-| Full date       | "yyyy'-'MM'-'dd"            | Calendar date                                                                   |
-| Full time       | "'Partial time'K"           | UTC of day or Local time of day with the time offset between local time and UTC |
-| Date time       | "'Full date''T''Full time'" | Calendar date and time of day, e.g. 2019-07-26T16:59:57-05:00                   |
+| Second fraction | "FFFFFFF"                   | Minimum of one digit, maximum of 16 digits.                                     |
+| Time offset     | "K"                         | Either "Z" or "('+'/'-')HH':'mm".                                               |
+| Partial time    | "HH':'mm':'ss[FFFFFFF]"     | Time without UTC offset information.                                            |
+| Full date       | "yyyy'-'MM'-'dd"            | Calendar date.                                                                  |
+| Full time       | "'Partial time'K"           | UTC of day or Local time of day with the time offset between local time and UTC.|
+| Date time       | "'Full date''T''Full time'" | Calendar date and time of day, for example, 2019-07-26T16:59:57-05:00.                  |
 
 ### Support for parsing
 
 The following levels of granularity are defined for parsing:
 
-1. 'Full date'
-    1. "yyyy'-'MM'-'dd"
+1. `'Full date'`
```

Comment:
On further review, I would say it's safer to keep the quotes, as changing to code format obscures the intention behind the string formatting.  For example:
"yyyy'-'MM'-'dd"
Because of the relative placement of double and single quotes, it's clear that yyyy, MM, and dd are the variables and '-' is unchanging punctuation. When you make the whole string in code format and remove the double quotes it's no longer clear what function the single quotes have; they become part of the code format string. So I would say we should retain the original use of quotes throughout this article to avoid causing confusion. Thanks to @gewarren for calling my attention to the issue.

Reply:
Ok, retained the original content.